packages/features/troubleshooter/store.ts

```diff
@@ -0,0 +1,110 @@
+import { useEffect } from "react";
+import { create } from "zustand";
+
+import dayjs from "@calcom/dayjs";
+
+import { updateQueryParam, getQueryParam, removeQueryParam } from "../bookings/Booker/utils/query-param";
```

Comment:
Using the same utils as booker for query params 

This troubleshooter is infact an atom because it was just as easy to do that